src/main/main.cpp

```diff
@@ -71,11 +71,25 @@ unique_ptr<GPU> create_gpu(MemoryInterface *memory_ptr, CPUInterface *cpu_ptr,
 int main(int argc, char *argv[]) {
 	ios_base::sync_with_stdio(false);
 
+	auto rom_path = string("");
 	if (argc < 2) {
-		cerr << "Please provide the name of a ROM file." << endl;
+		cerr << "Please provide the name of a ROM file or press q to exit" << endl;
+		getline(cin, rom_path);
+		if(rom_path == "q"){
+			cout << "Exiting TVP" << endl;
+			exit(1);
```

Comment:
I think an `exit(0)` would work better here, since this is not an error exit and is expected by the user, and hence a 0 exit status scenario.